app/components/SpecialDay/AprilFools/index.js

```diff
@@ -0,0 +1,96 @@
+// @flow
+import React from 'react';
+import styles from './AprilFools.css';
+import moment from 'moment-timezone';
+import { IndexLink } from 'react-router';
+import { Image } from 'app/components/Image';
+import logoImage from 'app/assets/logo-dark.png';
+import foolsImage from 'app/assets/aprilfools-image.png';
+import LoadingBox from './LoadingBox';
+
+type Props = {
+  children: Node
+};
+
+type State = {
+  hasClosed: boolean
+};
+
+class AprilFools extends React.Component<Props, State> {
+  state = {
+    hasClosed: false
+  };
+  static isCorrectDate() {
+    // The year doesn't matter, as it only check month and day of the month
+    //const aprilFoolsDate = moment('2019-04-01');
+    const aprilFoolsDate = moment('2019-01-28');
+    const now = moment();
+    return (
+      aprilFoolsDate.date() === now.date() &&
+      aprilFoolsDate.month() === now.month()
+    );
+  }
+
+  onQueueDone() {
+    localStorage.setItem('hasClosedAprilFools', 'true');
+    this.setState({
+      hasClosed: true
+    });
+  }
+
+  componentDidMount() {
```

Comment:
Since the state is being set after mount, will the popup "blink" (show and then disappear) every time the user reloads the page?

If that's the case, maybe it's better to use `hasClosed: true` as the initial value?